airflow/contrib/operators/qubole_operator.py

```diff
@@ -5,96 +5,82 @@
 
 class QuboleOperator(BaseOperator):
     """
-    Executes commands on Qubole (https://qubole.com).
+    Execute tasks (commands) on QDS (https://qubole.com).
+
+    :param qubole_conn_id: Connection id which consists of qds auth_token
```

Comment:
I'd it inside `init` only, but docs was not reflecting them. Infact all other operators from contrib which having docstring inside `init` not showing up. Operators like `SSHExecuteOperator` or `HipChatAPIOperator` have docstring right after class definition and being reflected. Not sure if something wrong with my setup only.